src/read.c

```diff
@@ -1284,21 +1289,20 @@ static avifBool avifDecoderGenerateImageGridTiles(avifDecoder * decoder, avifIma
 // dstImage. Only the freshly decoded tiles are considered, skipping the already
 // copied or not-yet-decoded tiles.
 static avifBool avifDecoderDataFillImageGrid(avifDecoderData * data,
-                                             avifImageGrid * grid,
                                              avifImage * dstImage,
-                                             unsigned int firstTileIndex,
+                                             const avifTileGroup * tileGroup,
```

Comment:
I do not follow. The name of the function describes its effect, not the arguments. I do not see a cost here. A shorter signature is even a benefit to me.

Reply:
This isn't that important. I was just stating my first impression when I had only skimmed the diffs quickly. What I saw was:

Before the change:
```
static avifBool avifDecoderDataFillImageGrid(avifDecoderData * data,
                                             avifImageGrid * grid,
```

Notice how the function name "avifDecoderDataFillImageGrid" matches the types of the first two parameters. This matching is gone after the change.

Now, looking at this more, I think the original matching was actually misleading, because the "FillImageGrid" part of the function name actually refers to "fill `avifImage * dstImage`" rather than "fill `avifImageGrid * grid`".